projects/jump_ci/testing/test.py

```
@@ -111,8 +110,11 @@ def prepare_env_file(_extra_env):
         run.run_toolbox("jump_ci", "ensure_lock", cluster=cluster, owner=utils.get_lock_owner())
 
         cluster_lock_dir = f" /tmp/topsail_{cluster}"
+        config_test_args
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)